doc/Doxyfile

```diff
@@ -879,7 +865,7 @@ IGNORE_PREFIX          =
 # If the GENERATE_HTML tag is set to YES (the default) Doxygen will
 # generate HTML output.
 
-GENERATE_HTML          = YES
```

Comment:
why change this? you call make html that will trigger html, so this is a strange change.